bin/setup

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+# Dispatches each module's setup script
+
+# Configure the script to be verbose and fail early
+# See: https://ss64.com/bash/set.html
+# See: https://bash.cyberciti.biz/guide/$IFS
+set -euo pipefail
+IFS=$'\n\t'
+set -vx
+
+# Seam to conditionalize different behavior in CI and local
```

Comment:
Not: It feels like this is unnecessary here; since it's not actually dispatching anything differently. I could see the benefit of leaving it as well.